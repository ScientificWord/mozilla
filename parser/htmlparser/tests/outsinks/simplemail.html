<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<body>This is a mail
 with a couple of long lines and then a sig. This is used as test of the
 format=flowed output in the nsHTMLToTXTSinkstream. If this test fails and
 none else, it's likely the spaces at the ends of the lines that are missing.
 They aren't easily seen without looking at the data in an editor and checking
 where the end of line is.<br>
<br>
Testing Space stuffing<br>
From shouldn't start a line.<br>
&nbsp; Nor a space.<br>
&gt; Neither should a greater-than symbol.<br>
--<br>
Above is NOT a sig delimiter since it's missing the trailing SPACE:<br>
<br>
-- <br>
Above is a sig delimiter. I hope.<br>
A sig (wow!).&nbsp; </body>
