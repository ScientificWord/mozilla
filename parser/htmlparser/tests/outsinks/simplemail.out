This is a mail with a couple of long lines and 
then a sig. This is used as test of the 
format=flowed output in the nsHTMLToTXTSinkstream. 
If this test fails and none else, it's likely the 
spaces at the ends of the lines that are missing. 
They aren't easily seen without looking at the 
data in an editor and checking where the end of 
line is.

Testing Space stuffing
 From shouldn't start a line.
  Nor a space.
 > Neither should a greater-than symbol.
--
Above is NOT a sig delimiter since it's missing 
the trailing SPACE:

-- 
Above is a sig delimiter. I hope.
A sig (wow!). 
